instrumentation/jodd-http-4.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/joddhttp/v4_2/JoddHttpHttpAttributesGetter.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.joddhttp.v4_2;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpClientAttributesGetter;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+import jodd.http.HttpRequest;
+import jodd.http.HttpResponse;
+
+final class JoddHttpHttpAttributesGetter
+    implements HttpClientAttributesGetter<HttpRequest, HttpResponse> {
+  private static final Logger logger =
+      Logger.getLogger(JoddHttpHttpAttributesGetter.class.getName());
+
+  @Override
+  public String getMethod(HttpRequest request) {
+    return request.method();
+  }
+
+  @Override
+  public String getUrl(HttpRequest request) {
+    return request.url();
+  }
+
+  @Override
+  public List<String> getRequestHeader(HttpRequest request, String name) {
+    return request.headers(name);
+  }
+
+  @Override
+  public Integer getStatusCode(
+      HttpRequest request, HttpResponse response, @Nullable Throwable error) {
+    return response.statusCode();
+  }
+
+  @Override
+  @Nullable
+  public String getFlavor(HttpRequest request, @Nullable HttpResponse response) {
+    String httpVersion = request.httpVersion();
+    if (httpVersion == null && response != null) {
+      httpVersion = response.httpVersion();
+    }
+    if (httpVersion != null) {
+      if (httpVersion.contains("/")) {
+        httpVersion = httpVersion.substring(httpVersion.lastIndexOf("/") + 1);
+      }
+
+      if (httpVersion.equals(SemanticAttributes.HttpFlavorValues.HTTP_1_0)) {
```

Comment:
Static import for readability?

Reply:
Added static imports for better readability